frontend/src/app/projects/project-detail/project-users/project-user-settings.component.ts

```diff
@@ -70,7 +70,8 @@ export class ProjectUserSettingsComponent implements OnChanges {
   permissionRequiredValidator(): ValidatorFn {
     return (control: AbstractControl): ValidationErrors | null => {
       if (
-        control.get('permission')?.value in this.repoUserService.PERMISSIONS ||
+        control.get('permission')?.value in
```

Comment:
For me, it would be better here to use `let permission = control.get('permission')` and `let role = control.get('role')` instead of repeating them multiple times.

Reply:
Done.